src/lib/consensus/proof_of_stake.ml

```diff
@@ -523,35 +523,6 @@ module Make (Inputs : Inputs_intf) :
 
     let genesis =
       {hash= genesis_ledger_hash; total_currency= genesis_ledger_total_currency}
-
-    let get_stake ~logger ~public_key ~local_state {hash; total_currency} =
-      let open Local_state in
-      let open Option.Let_syntax in
-      let%bind ledger =
-        if Coda_base.Frozen_ledger_hash.equal hash genesis_ledger_hash then
```

Comment:
This check for which ledger to access is missing now, isn't it? Wherever we grab the ledger for looking up vrf evaluations will need to handle this genesis ledger case. It should also probably assert the hash equality in this else branch as well.

Reply:
oo good call -- I will fix now

Reply:
ok think I've done it